CMakeLists.txt

```diff
@@ -431,10 +431,10 @@ if(NOT "${CMAKE_GENERATOR_TOOLSET}" STREQUAL "")
 endif()
 add_custom_target(test_install
     COMMAND ${CMAKE_COMMAND} -G${CMAKE_GENERATOR} ${TOOLSET}
-    -H${GINKGO_TEST_INSTALL_SRC_DIR}
+    -S${GINKGO_TEST_INSTALL_SRC_DIR}
```

Comment:
I think this might have been some CMake argument parsing weirdness, because -H is a flag without argument, the "next one"/following characters might have been parsed as a positional argument, which by default is used for the source path.